.github/workflows/performance-monitoring.yml

```
@@ -0,0 +1,601 @@
+          EOF
+          
+          node compare-performance.js > performance-report.md
+          
+      - name: Comment PR with performance report
+        if: github.event_name == 'pull_request'
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const fs = require('fs');
+            const report = fs.readFileSync('performance-report.md', 'utf8');
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)